zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterDownloader.java

```diff
@@ -51,22 +50,21 @@ public RemoteInterpreterDownloader(
   }
 
   public static void main(String[] args) {
-    String zeppelinServerHost = null;
-    int port = Constants.ZEPPELIN_INTERPRETER_DEFAUlT_PORT;
-    String interpreter = null;
-    String localRepoPath = null;
-    if (args.length > 0) {
-      zeppelinServerHost = args[0];
-      port = Integer.parseInt(args[1]);
-      interpreter = args[2];
-      localRepoPath = args[3];
-    }
-    RemoteInterpreterEventClient intpEventClient = new RemoteInterpreterEventClient(
-        zeppelinServerHost, port, 3);
+    if (args.length == 4) {
+      String zeppelinServerHost = args[0];
+      int port = Integer.parseInt(args[1]);
+      String interpreter = args[2];
+      String localRepoPath = args[3];
+      RemoteInterpreterEventClient intpEventClient = new RemoteInterpreterEventClient(
+          zeppelinServerHost, port, 3);
 
-    RemoteInterpreterDownloader downloader = new RemoteInterpreterDownloader(interpreter,
-        intpEventClient, new File(localRepoPath));
-    downloader.syncAllLibraries();
+      RemoteInterpreterDownloader downloader = new RemoteInterpreterDownloader(interpreter,
+          intpEventClient, new File(localRepoPath));
+      downloader.syncAllLibraries();
+    } else {
+      LOGGER.error(
+          "Wrong amount of Arguments. Expected: [ZeppelinHostname] [ZeppelinPort] [InterpreterName] [LocalRepoPath]");
+    }
```

Comment:
In this case, the script executing this program cannot determine that this program has not been executed normally due to the mismatch of the arguments. Is this intentional?

Reply:
Thanks for your review, an exit code has been added.